compiler-core/src/format.rs

```diff
@@ -1585,10 +1586,9 @@ impl<'comments> Formatter<'comments> {
                     },
                 )
                 .collect_vec();
-            constructor
-                .name
-                .as_str()
-                .to_doc()
+
```

Comment:
I'm having issues getting the formatting to work here.
For example, the following is formatting to this 
```gleam
type Fooing {
  @deprecated("")
  Bar(//test
  a: Int, b: Int)
  Foo
}
```
instead of:
```gleam
type Fooing {
  @deprecated("")
  Bar(
    //test
    a: Int,
    b: Int,
  )
  Foo
}
```